src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,97 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.exceptions.ValidationException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private RegistrationService registrationService = new RegistrationServiceImpl();
+    private User firstUser = new User();
+    private User secondUser = new User();
+
+    @BeforeEach
+    void setUp() {
+        Storage.people.clear();
+        firstUser.setLogin("First");
+        firstUser.setAge(27);
+        firstUser.setPassword("12345678");
+        secondUser.setLogin("Second");
+        secondUser.setAge(29);
+        secondUser.setPassword("87654321");
+    }
+
+    @Test
+    void register_validUserAdd_Ok() {
+        registrationService.register(firstUser);
+        registrationService.register(secondUser);
+        int actual = Storage.people.size();
+        assertEquals(2, actual);
+    }
+
+    @Test
+    void register_userAge18_Ok() {
+        firstUser.setAge(18);

```

Comment:
```suggestion
        firstUser.setAge(RegistrationServiceImpl.MIN_USER_AGE);

```

and we should make this constant public